DigitalLearningSolutions.Data.Tests/Services/LearningHubLinkServiceTests.cs

```diff
@@ -0,0 +1,261 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using System;
+    using DigitalLearningSolutions.Data.DataServices.UserDataService;
+    using DigitalLearningSolutions.Data.Exceptions;
+    using DigitalLearningSolutions.Data.Models.Signposting;
+    using DigitalLearningSolutions.Data.Services;
+    using FakeItEasy;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class LearningHubLinkServiceTests
+    {
+        private ILearningHubLinkService learningHubLinkService = null!;
+        private ILearningHubSsoSecurityService learningHubSsoSecurityService = null!;
+        private IUserDataService userDataService = null!;
+
+        [SetUp]
+        public void Setup()
+        {
+            userDataService = A.Fake<IUserDataService>();
+            learningHubSsoSecurityService = A.Fake<ILearningHubSsoSecurityService>();
+            A.CallTo(() => learningHubSsoSecurityService.VerifyHash("56789", A<string>._)).Returns(false);
+            A.CallTo(() => learningHubSsoSecurityService.VerifyHash("12345", A<string>._)).Returns(true);

```

Comment:
Might be nice to change these to "invalid-hash" and "valid-hash" respectively to make the contents of the tests a bit more understandable, or just set it up in each test.

Reply:
Yeah makes sense, done.